candies/cli/parsers/standard.py

```diff
@@ -103,37 +131,53 @@ def configured(new: Callable, command: Command, prefix: str = '.') \
         {'.command': 'tool', '..command': 'install', 'g': 'something'}.
     """
 
+    signature = inspect.signature(command.func)
     parser = new(command.name,
                  description=command.description,
-                 formatter_class=help(command))
+                 formatter_class=help(command),
+                 prefix_chars=prefixes(signature))
 
-    def short_of(x: Any) -> Optional[str]:
-        short = getattr(x, 'short', None)
+    def short_of(x: inspect.Parameter, kind: Any) -> Optional[str]:
+        short = getattr(kind, 'short', None)
 
         if short is None:
             # If `None`, then considered as omitted.
             return short
 
         if short == '':
-            raise ValueError(f"Flag '{parameter.name}' has invalid short name "
+            raise ValueError(f"Argument '{x.name}' has invalid short name "
                              f"'{short}': "
                              f"it is an empty string")
 
         if len(short) > 1:
-            raise ValueError(f"Flag '{parameter.name}' has invalid short name "
+            raise ValueError(f"Argument '{x.name}' has invalid short name "
                              f"'{short}': "
                              f"it consists of more than one character")
 
         if not short.isalpha():
-            raise ValueError(f"Flag '{parameter.name}' has invalid short name "
+            raise ValueError(f"Argument '{x.name}' has invalid short name "
                              f"'{short}': "
                              f"it is not an alphabet character")
 
         return short
 
-    def prefixes_of(x: Any) -> Tuple[str, str]:
-        # TODO: Add more prefixes depending on `prefix` attribute.
-        return '-', '--'
+    def prefixes_of(x: inspect.Parameter, kind: Any) -> Tuple[str, str]:
+        prefix = getattr(kind, 'prefix', None)
+
+        if prefix is None:
+            return '-', '--'
+
+        if prefix == '':
+            raise ValueError(f"Argument '{x.name}' has invalid prefix "
```

Comment:
There is an article missing: "has an invalid prefix".

Reply:
Fixed.